src/sidebar/services/groups.ts

```diff
@@ -478,4 +480,71 @@ export class GroupsService {
       userid: 'me',
     });
   }
+
+  /**
+   * Fetch members for a group from the API and load them into the store as the
+   * members of the focused group.
+   *
+   * @param [signal] - If provided, it can be used to cancel the loading of
+   *                   group members when aborted
+   */
+  async loadFocusedGroupMembers(
+    groupId: string,
+    signal?: AbortSignal,
+  ): Promise<void> {
```

Comment:
The groups service could handle cancellation of the request itself in the `focus` method, which is always called to trigger a group change. It also knows what the focused group is, so the API for this method could just be `loadFocusedGroupMembers()`.